### PR TITLE
Submit form

### PR DESCRIPTION
**Pull request by KaiElwood**
Waiting on Brian's component to fully finish this... could just pull from main and implement it, but wanted to push this up earlier for review so ya'll can see how the other components would fit with this one

### CODE REVIEW

File: front-end/src/SubmitForm/SubmitForm.jsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+		}
+		setSelectedFile(e.target.files[0])
+	}
+
+	return (
+		<div>
+			<form className="form-wrapper" onSubmit={handleSubmit(onSubmit)}>
+				<div className="form-control">
+					<div className="imgContainer">
+						{preview && <img alt="Stoop" src={preview} />}
```
Comment (163dgm) [inline front-end/src/SubmitForm/SubmitForm.jsx:57]: Can you put a fall back image/ icon that shows when there's no preview

File: front-end/src/SubmitForm/SubmitForm.jsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+					</label>
+				</div>
+				<div className="form-control">
+					<label
+						htmlFor="title"
+						className="input-group input-group-vertical"
+					>
+						<span>Title</span>
+						<input
+							id="title"
```
Comment (163dgm) [inline front-end/src/SubmitForm/SubmitForm.jsx:84]: Can you give your id's more unique names?

File: front-end/src/SubmitForm/SubmitForm.jsx [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+							name="location"
+							{...register('location', {
+								required: 'Location is required.'
+							})}
+						/>
+					</label>
+					{errors.location && (
+						<p className="errorMsg">{errors.location.message}</p>
+					)}
+					{/* this will be merged with Brian's code, which already provides these methods */}
```
Comment (163dgm) [inline front-end/src/SubmitForm/SubmitForm.jsx:138]: Brian's code is done now, is this merged?

**Review by shl622 — APPROVED** [review]

**Review by shl622 — APPROVED** [review]
looks clean

**Review by coherency1 — APPROVED** [review]
need second opinion on map components but looks good to me

**Pull request merged**